waspc/data/Generator/templates/server/src/webSocket/initialization.ts

```diff
@@ -43,7 +44,10 @@ async function addUserToSocketDataIfAuthenticated(socket: Socket, next: (err?: E
   if (sessionId) {
     try {
       const { user } =  await getSessionAndUserFromSessionId(sessionId)
-      socket.data = { ...socket.data, user }
+      socket.data = {
+        ...socket.data,
+        user: user ? enrichAuthUser(user) : null
```

Comment:
This null thing repeats a couple of times. What do you think about extracting it into a function?

Reply:
Hm, I wonder what to call it. `enrichAuthUserIfExists` or something like that?